bindings/dart/lib/src/web/reranker/data_provider.dart

```diff
@@ -4,8 +4,20 @@ import 'package:xayn_ai_ffi_dart/src/common/reranker/data_provider.dart'
     as common show Asset, AssetType, baseAssets, SetupData;
 
 /// Returns a map of all assets required for initializing [`XaynAi`].
-Map<common.AssetType, common.Asset> getAssets() {
-  return common.baseAssets;
+Map<common.AssetType, common.Asset> getAssets(
+    {List<String> features = const []}) {
+  final redundantAssets = features.contains('webParallel')
```

Comment:
2 minor nit pickings which you can totally ignore if you want :wink: 

1. I personally prefer `webThreads` there are other forms of parallelism we could have implemented (but which would be x10-x100 more work to implement with **much** less speedups, etc. So it's not really an option, but because of this maybe going with `webThreads ` is better, but it doesn't really matter I guess)

2. If you feel fancy you could change `List<String>` to `Set<FeatureHint>` with `enum FeatureHint { webThreads, ...}` but it doesn't really matter, tbh.

Reply:
we decided to user `parallel`to be consistent with the naming in rust.

Reply:
> I personally prefer webThreads there are other forms of parallelism we could have implemented (but which would be x10-x100 more work to implement with much less speedups, etc. So it's not really an option, but because of this maybe going with webThreads is better, but it doesn't really matter I guess)

i thought of it more like `webParallel` subsumes multiple things, for now only multi-threading, and later for example also simd. if we ever need a more fine-grained feature selection we can split it, but it's probably not needed.

> If you feel fancy you could change List<String> to Set<FeatureHint> with enum FeatureHint { webThreads, ...} but it doesn't really matter, tbh.

i'm going to change this as @acrrd requested it as well.